src/Api/Models/Request/Accounts/KdfRequestModel.cs

```diff
@@ -9,6 +9,8 @@ public class KdfRequestModel : PasswordRequestModel, IValidatableObject
     public KdfType? Kdf { get; set; }
     [Required]
     public int? KdfIterations { get; set; }
+    public int? KdfMemory { get; set; }
+    public int? KdfParallelism { get; set; }
```

Comment:
Do we need any validation on these, similar to the below for PBKDF2?

Reply:
Yeah, it is more of a concept-pr for initial review, but you're right. What are some sensible values here?

Going by OWASP: https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html
the minimum configuration should be 15MiB of memory, iteration count of 2, and 1 degree of parallelism. In my opinion we can set the defaults in the clients a bit higher than this, but this should be the minimum. For reference, in the WebAssembly version, on Firefox on my laptop it takes ~70ms to run this minimum (40ms with the SIMD implementation). 
On my phone, in chrome this takes ~290ms (130ms with the SIMD implementation).
Definitely short enough to where the user does not notice this significantly.

As for maximum:
I'm not sure what we want to limit here. Protect the user from too large runtime?
For memory: On both chrome and firefox, the argon2-browser library works up to 1GiB in memory cost. Above that I get a memory allocation error. At 1GiB and 2 iterations it takes ~4 seconds (2 seconds with the SIMD implementation).
So if we want to protect the user from too large runtimes, we need to take both memory and iterations into account.

Reply:
Added validation:

https://github.com/bitwarden/server/blob/29af2b3ef7693b42c9cf307c5ef8f146b8d42d3e/src/Api/Models/Request/Accounts/KdfRequestModel.cs#L27-L41